Automattic/pocket-casts-ios
# Playback 2024: Sharing

| 📘 Part of: #2250 |
|:---:|

Completes #2373


| | | | |
|----------|----------|----------|----------|
| ![IMG_0056](https://github.com/user-attachments/assets/fb4d03e1-2283-4a05-a9aa-daacfbd3200e) | ![IMG_0057](https://github.com/user-attachments/assets/a718372c-2049-41eb-a166-57da4613edff) | ![IMG_0058](https://github.com/user-attachments/assets/5a963bff-c8aa-46c4-a8e8-1b374a7ea2c8) | ![IMG_0059](https://github.com/user-attachments/assets/458e5ad7-8458-4d3b-b2d7-b768a5503e9e) |
| ![IMG_0060](https://github.com/user-attachments/assets/f50afcaa-a5e5-489b-bc37-f13fff0caa64) | ![IMG_0061](https://github.com/user-attachments/assets/502b723e-6ed8-4271-9e0d-f8b1768ea9e7) | ![IMG_0062](https://github.com/user-attachments/assets/f1578457-77cd-4ea4-ab63-07ce87ddba58) | ![IMG_0063](https://github.com/user-attachments/assets/618babab-b0c0-41bf-945b-7154657e5689) |

## To test

* Enable the `endOfYear2024` feature flag
* Restart the app
* Navigate to Profile and tap the "Playback 2024" cell
* Go through each screen and tap the "Share this story" button
* Verify the saved image
* Verify the shared text matches the screen

## Checklist
